AppIntro.js

```diff
@@ -188,14 +188,72 @@ export default class AppIntro extends Component {
     };
   }
 
+  renderSkipButton = () => {
+    if (this.props.renderSkipButton) {
```

Comment:
this.props.renderSkipButton ?
I see you props are defined `showSkipButton`
 `renderDoneButton` also has the same problem